.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.aarch64-unknown-linux-musl]
```

Comment:
nit: typo in commit message 9914baec3fe34b7dcfbb5757abf7517a1cae40dc: s/flotaing/floating